CMakeLists.txt

```diff
@@ -272,6 +272,11 @@ if(ENABLE_PAM)
 endif()
 set(HAVE_PAM ${ENABLE_PAM})
 
+# Check for MIT-SHM
+if(X11_XShm_FOUND)
+    add_definitions("-DHAVE_MITSHM")
```

Comment:
We already assume MITSHM in vncviewer, so I suggest simply removing the `#ifdef`s.